src-appsetting/_MicroserviceTemplate_.WebApi/appsettings-for-using-env.json

```diff
@@ -0,0 +1,3 @@
+{
+   "UseEnvironmentVariables": true  
```

Comment:
@parimal1987 why this?  src-msdb, NO need for this use env

Reply:
@Joseph1977 In the initial commit you have added this file in the old template and I used the same branch for the development.
Let me remove it.